Assignment/training3/src/features/pages/Home/Home.js

```diff
@@ -0,0 +1,34 @@
+import React, {useState} from 'react';
+import Modal from '../../../components/Modal/Modal';
+import './Home.css';
+
+
+const Home = () => {
+
+    const [hideModal, setHideModal] = useState(false);
+
+    const openModal = (e) => {
+        setHideModal(!hideModal);
+    }
+
+    return (
+        <div>
+            { !hideModal ?
```

Comment:
@lehuy1012 As i said, the modal should be design like other modal lib, we can use condition to hide the modal while the modal has the prop hide

Reply:
fixed